tests/conftest.py

```diff
@@ -42,6 +42,84 @@ def create_yaml_workflow_schema():
     return reana_yaml_schema
 
 
+@pytest.fixture()
+def create_cwl_yaml_workflow_schema():
+    """Return dummy cwl workflow schema."""
+    reana_cwl_yaml_schema = """
+        version: 0.7.2
+        workflow:
+          type: cwl
+          file: test.cwl
+        outputs:
+          files:
+            - foo/bar
+        """
+    return reana_cwl_yaml_schema
+
+
+@pytest.fixture()
+def cwl_workflow_spec_correct_input_param():
+    """Return correct dummy CWL workflow loaded spec."""
+    cwl_workflow_spec = """
+        #!/usr/bin/env cwl-runner
+
+        arguments:
+        - -c
+        baseCommand: bash
+        class: CommandLineTool
+        cwlVersion: v1.0
+        id: foo
+        inputs:
+        - id: foo
+          inputBinding:
+            prefix: --inp-foo=
+            separate: false
+          type: string
+        - id: bar
+          inputBinding:
+            prefix: --inp-bar=
+            separate: false
+          type: int
+        outputs:
+        - doc: fooBar
+          id: summary
+          type: string
+        requirements:
+          DockerRequirement:
+            dockerPull: foo/bar
+        """
+    return cwl_workflow_spec
+
+
+@pytest.fixture()
+def cwl_workflow_spec_wrong_input_param():
+    """Return wrong dummy CWL workflow loaded spec."""
+    cwl_workflow_spec = """
+        #!/usr/bin/env cwl-runner
+
+        arguments:
+        - -c
+        baseCommand: bash
+        class: CommandLineTool
+        cwlVersion: v1.0
+        id: foo
+        inputs:
+        - id: bar
+          inputBinding:
+            prefix: --inp-bar=
+            sepearate: false
```

Comment:
Is this a typo? I still don't understand what's wrong with this spec. Where are the steps?

Reply:
Yes, This is the intentional error to check.

Reply:
Hard to catch... I think this isn't validating what we want... It's validating syntax instead of parameters. What we want to catch is that you have e.g. a parameter called `foo` and you reference it with `fooo`.